src/uwtools/resources/jsonschema/execution.jsonschema

```diff
@@ -4,6 +4,9 @@
     "batchargs": {
       "additionalProperties": true,
       "properties": {
+        "threads": {
+          "not": {}
+        },
```

Comment:
This forbids `threads` under `batchargs`. This should not be set directly: It will be injected by the driver code based on the presence and value of the `execution.threads` item.